src/FlaUI.Core/Logging/DebugLogger.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace FlaUI.Core.Logging
+{
+    public class DebugLogger : AbstractBaseLogger
+    {
+        public override void GatedDebug(string message)
+        {
+            System.Diagnostics.Debug.WriteLine(message);
```

Comment:
How about adding a prefix like INFO, ERROR, WARN so that the errorlevel is visible in the message?